metals/src/main/scala/scala/meta/internal/metals/codeactions/CodeAction.scala

```diff
@@ -19,7 +20,7 @@ trait CodeAction {
   def kind: String
 
   type CommandData
-  type ActionCommand = ParametrizedCommand[CommandData]
+  type ActionCommand = CodeActionCommand[CommandData]
```

Comment:
To be fair, I think I can simplify this and do it for the focused file again. I didn't want to abuse it since I wasn't sure if it won't cause issues in some edge cases (e.g. if something goes wrong and the focused file is incorrect, all goes wrong). And passing the arg felt safer but maybe it'd be okay. What do you reckon?